instruments/thorlabs/thorlabsapt.py

```diff
@@ -282,7 +282,9 @@ def max_travel(self):
                 source=0x01,
                 data=None
             )
-            resp = self._apt.querypacket(pkt)
+            resp = self._apt.querypacket(
+                pkt, expect_data_len=4
+            )
```

Comment:
W/o an `expect_data_len` argument, the unpacking does not return data. Ensured throughout all additions of `expect_data_len` throughout that the expected data length agrees with latest manual.